src/ngraph_mark_for_clustering.cc

```diff
@@ -62,18 +61,6 @@ static Status NGraphPlacementRequested(Node* node, bool& placement_ok) {
   return Status::OK();
 }
 
-#if !defined(NGRAPH_TF_DISABLE_DEADNESS_CHECK)
-// Checks if the node's inputs have mismatching deadness
-static Status DeadnessOk(Node* node,
-                         std::unique_ptr<DeadnessAnalysis>* deadness_analyzer,
-                         bool& deadness_ok) {
-  deadness_ok =
-      !(node->IsMerge() ||
```

Comment:
I know this is removed, but just curious as to why we were only checking for IsMerge() and not IsSwitch etc.?

Reply:
This is a requirement of HasInputsWithMismatchingDeadness() api. It is designed to take only non-merge ops.  It fails if the node is a merge op.